shared/iot/develop/_ensure-channel-quality.mdx

```diff
@@ -44,7 +46,10 @@ To provide the best audio and video quality in your <Vpl k="CLIENT"/>:
 The following figure shows the workflow you need to implement to ensure channel quality in your <Vpl k="CLIENT"/>:
 
 ![Ensure Channel Quality](/images/iot/iot-channel-quality.svg)
-
+<PlatformWrapper platform="linux-c">
+**This page is coming soon for Linux c.**
+</PlatformWrapper>
+<PlatformWrapper notAllowed="linux-c">
 ## Prerequisites
```

Comment:
Two questions. 
1. Does it make sense to say that this page is coming soon for Linux only after the whole intro and diagram?
2. The variable for "app" or whatever it is does not show in text on the Linux page:
![Screenshot](https://user-images.githubusercontent.com/114177030/220926573-cc7db294-f8c3-49d8-8634-e5b54d72c6f6.png)
Same for other doc pages. For example, see "Secure channel encryption"

Reply:
1. Inro & diagam is there. rest I will covered with drafting page for "Channel Quality" in different PR.
2. It has been fixed in another PR.